client/test/e2e/inventory/inventory_configuration.spec.js

```diff
@@ -0,0 +1,112 @@
+/* global element, by, inject, browser */
+'use strict';
+
+const chai   = require('chai');
+const expect = chai.expect;
+
+const FU = require('../shared/FormUtils');
+const helpers = require('../shared/helpers');
+const components = require('../shared/components');
+
+helpers.configure(chai);
+
+describe.only('Inventory Configuration ::', () => {
+  'use strict';
+
+  // navigate to the page
+  before(() => helpers.navigate('#/inventory/configuration'));
+
+  const group = {
+    name : '[E2E] Inventory Group',
+    code : '1704',
+    sales_account : 'Test Capital One',
+    stock_account : 'Test Capital Two',
+    cogs_account  : 'Test Balance'
+  };
+
+  const updateGroup = {
+    name : '[E2E] Inventory Group updated',
+    code : '2504',
+    sales_account : 'Test Inventory Accounts',
+    stock_account : '',
+    cogs_account  : ''
+  };
+
+  // inventory type
+  const type = { text : '[E2E] Inventory Type' };
+  const updateType = { text : '[E2E] Inventory Type updated' };
+
+  // inventory unit
+  const unit = { text : '[E2E] Inventory Unit' };
+  const updateUnit = { text : '[E2E] Inventory Unit updated' };
+
+  describe('Groups ::', () => {
+    // navigate to the page
+    before(() => helpers.navigate('#/inventory/configuration'));
```

Comment:
Since all of these use the same page, you do not need to refresh the page by navigating to `/#/inventory/configuration` each time.  The first `before()` call will have done that.